release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -387,9 +394,9 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
 For containers interacting with SR-IOV virtual functions (VFs) in Data Plane Development Kit (DPDK) mode, the `app-netutil` library now provides the following functions: `GetCPUInfo()`, `GetHugepages()`, and `GetInterfaces()`. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#nw-sriov-app-netutil_about-sriov[DPDK library for use with container applications].
 
 [id="ocp-4-7-ovn-kubernetes-egress-router-cni"]
-==== Egress router CNI (Technical Preview)
+==== Egress router CNI (Technology Preview)
 
-The egress router CNI plug-in is introduced in Technical Preview. You can use the plug-in to deploy an egress router in redirect mode. This egress router provides parity for OVN-Kubernetes compared to OpenShift SDN, but for redirect mode only. The plug-in does not perform in HTTP proxy or DNS proxy modes, and this is a difference with the implementation for OpenShift SDN. For more information, see xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc[Deploying an egress router pod in redirect mode].
+The egress router CNI plug-in is introduced in Technology Preview. You can use the plug-in to deploy an egress router in redirect mode. This egress router provides parity for OVN-Kubernetes compared to OpenShift SDN, but for redirect mode only. The plug-in does not perform in HTTP proxy or DNS proxy modes, and this is a difference with the implementation for OpenShift SDN. For more information, see xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc[Deploying an egress router pod in redirect mode].
```

Comment:
.adoc#deploying-egress-router-ovn-redirection[Deploying an egress router pod in redirect mode].